libs/ui-watt/src/lib/components/dropdown/+storybook/storybook-dropdown-template-driven-forms.stories.ts

```diff
@@ -0,0 +1,178 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Meta, moduleMetadata, Story } from '@storybook/angular';
+import { FormsModule } from '@angular/forms';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+import { WattFormFieldModule } from '../../form-field/form-field.module';
+import { WattDropdownModule } from '../watt-dropdown.module';
+import { WattDropdownComponent } from '../watt-dropdown.component';
+import { WattDropdownOption } from '../watt-dropdown-option';
+
+const dropdownOptions: WattDropdownOption[] = [
+  { value: 'mightyDucks', displayValue: 'Mighty Ducks' },
+  { value: 'batman', displayValue: 'Batman' },
+  { value: 'titans', displayValue: 'Titans' },
+  { value: 'volt', displayValue: 'Volt' },
+  { value: 'joules', displayValue: 'Joules' },
+];
+
+export default {
+  title: 'Components/Dropdown/Template-driven Forms',
+  component: WattDropdownComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [
+        FormsModule,
+        BrowserAnimationsModule,
+        WattDropdownModule,
+        WattFormFieldModule,
+      ],
+    }),
+  ],
+} as Meta<WattDropdownComponent>;
+
+const howToUseGuideBasic = `
+ How to use
+ 
+ 1. Import ${WattDropdownModule.name} in a module
+ 
+ import { ${WattDropdownModule.name} } from '@energinet-datahub/watt';
+
+ 2a. Create model in a component for single selection
+ 
+ singleSelectionModel = '';
+
+ 2b. Create model in a component for multi selection
+ 
+ multiSelectionModel: string[] | null = null;
+ 
+ 3. Define dropdown options by using the WattDropdownOption interface
+ 
+ options: WattDropdownOption[] = [{ value: 'example', displayValue: 'Example' }]
+ 
+ 4. Assign the model and options to the dropdown component
+ 
+ <watt-dropdown [(ngModel)]="singleSelectionModel" [options]="options"></watt-dropdown>
+ 
+ 5. Wrap the dropdown component in a "watt-form-field"
+
+ <watt-form-field>
+  <watt-dropdown [(ngModel)]="singleSelectionModel" [options]="options"></watt-dropdown>
+ </watt-form-field>`;
+
+export const singleSelect: Story<WattDropdownComponent> = (
```

Comment:
Question: Should we have some showcases for `large` form-fields as well or is this out of scope for now?

Reply:
I think we should include it. It's also a good opportunity to test how the dropdown will work in a `large` form-field. Looks like Titans are only using `normal` form-fields for now. So we can make a follow-up PR with a showcase for `large` form-fields.

Reply:
Totally fine by me, do you create an issue for this follow-up for the backlog? :)

Reply:
Just created the issue in the backlog.